variants/t-echo/platformio.ini

```diff
@@ -11,6 +11,12 @@ build_flags = ${nrf52840_base.build_flags} -Ivariants/t-echo
   -DEINK_DISPLAY_MODEL=GxEPD2_154_D67
   -DEINK_WIDTH=200
   -DEINK_HEIGHT=200
+  -D USE_EINK_DYNAMICDISPLAY            ; Enable Dynamic EInk
```

Comment:
Maybe remove the space to be consistent with the rest.

Reply:
Ha, I did spot this before, and it did bother me; no objections here! (7275c21f6b054e57a9c2c1b8a418cb1aa047bab5)